[2025-01-24T23:18:18.085+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-24T23:18:18.102+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: executing_multiple_tasks.taskD manual__2025-01-24T23:17:59.400510+00:00 [queued]>
[2025-01-24T23:18:18.110+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: executing_multiple_tasks.taskD manual__2025-01-24T23:17:59.400510+00:00 [queued]>
[2025-01-24T23:18:18.114+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-01-24T23:18:18.131+0000] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): taskD> on 2025-01-24 23:17:59.400510+00:00
[2025-01-24T23:18:18.139+0000] {standard_task_runner.py:72} INFO - Started process 1391 to run task
[2025-01-24T23:18:18.141+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'executing_multiple_tasks', 'taskD', 'manual__2025-01-24T23:17:59.400510+00:00', '--job-id', '19', '--raw', '--subdir', 'DAGS_FOLDER/execute_multiple_tasks.py', '--cfg-path', '/tmp/tmpiz8t7jv3']
[2025-01-24T23:18:18.144+0000] {standard_task_runner.py:105} INFO - Job 19: Subtask taskD
[2025-01-24T23:18:18.188+0000] {task_command.py:467} INFO - Running <TaskInstance: executing_multiple_tasks.taskD manual__2025-01-24T23:17:59.400510+00:00 [running]> on host 6eaa7309f7d4
[2025-01-24T23:18:18.264+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='executing_multiple_tasks' AIRFLOW_CTX_TASK_ID='taskD' AIRFLOW_CTX_EXECUTION_DATE='2025-01-24T23:17:59.400510+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-24T23:17:59.400510+00:00'
[2025-01-24T23:18:18.266+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-24T23:18:18.278+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-01-24T23:18:18.280+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo TASK D completed!']
[2025-01-24T23:18:18.291+0000] {subprocess.py:99} INFO - Output:
[2025-01-24T23:18:18.293+0000] {subprocess.py:106} INFO - TASK D completed!
[2025-01-24T23:18:18.294+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-01-24T23:18:18.319+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-24T23:18:18.321+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=executing_multiple_tasks, task_id=taskD, run_id=manual__2025-01-24T23:17:59.400510+00:00, execution_date=20250124T231759, start_date=20250124T231818, end_date=20250124T231818
[2025-01-24T23:18:18.357+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-24T23:18:18.372+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-01-24T23:18:18.377+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
